practice-02/src/index.html

```diff
@@ -17,18 +17,115 @@
         <section id="box-filter">
             <!-- Filter -->
             <div class="block-filter-main">
-                <div class="filter-total">
+                <!-- Filter Total -->
+                <div class="filter-item filter-total">
                     <div class="filter-item__title">Filter</div>
+                    <div class="filter-show show-total">
+                        <div class="show-total-main">
+                            <div class="show-total-item clearfix">
+                                <div class="show-total-title">Manufacture</div>
+                                <div class="filter-list ">
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/logo-iphone-220x48.png" width="30"
```

Comment:
Miss alt. Please update ALL similar CASES

Reply:
Yes, I have updated the alt for all image cases